internal/server/backup/backup_bucket.go

```diff
@@ -0,0 +1,128 @@
+package backup
```

Comment:
Tiny naming nit, we usually replace `internal/server` with `incusd` in commit messages, so this one should be `incusd: Add storage bucket backup`